src/config.rs

```diff
@@ -73,6 +73,13 @@ configuration_option_enum! { Density:
     CompressedIfEmpty,
 }
 
+configuration_option_enum! { TypeDensity:
+    // No spaces around "=" and "+"
+    Compressed,
+    // Spaces around " = " and " + "
+    WhiteSpace,
```

Comment:
nit: I am not sure `WhiteSpace` is a great name. What about `Sparse`, `Spacious`, `Wide` or even `Normal`?